### ISSUE TITLE
If we do not specify the transition property (transform I guess) does it then default to all?

### ISSUE THREAD

**Issue by areblk** [scania-digital-design-system/sdds#327]
If we do not specify the transition property (transform I guess) does it then default to all?
If yes, see if it is possible to specify it, I think it is better for performance.

Same goes for line 317.

_Originally posted by @theJohnnyMe in https://github.com/scania-digital-design-system/sdds/pull/310#r808166547_

**Comment by nathalielindqvist** [comment]
Yes, if not specified it defaults to all changing properties. It is recommended to animate one or a couple changing properties at a time for the sake of performance, make sure to specify it in the transition. Something to keep in mind for future reference.
